stdpopsim/docs/_ext/speciescatalog.py

```diff
@@ -0,0 +1,622 @@
+"""
```

Comment:
Not sure why this file is in the pr as a complete file, rather than a diff. Something funny must be going on, but probably need to do a rebase at least.

Reply:
rebasing now

Reply:
Wow, this was confusing! This took me a while to spot, but you've accidentally copied the file: now in your PR are both
```
docs/_ext/speciescatalog.py
```
and
```
stdpopsim/docs/_ext/speciescatalog.py
```
Only the first version should be there. (And, the two files are identical, I think, so you can safely delete the second.)

Reply:
thanks! Deleted it!